crates/services/block_aggregator_api/src/db.rs

```diff
@@ -0,0 +1,20 @@
+use crate::{
+    blocks::Block,
+    result::Result,
+};
+use fuel_core_services::stream::BoxStream;
+
+pub trait BlockAggregatorDB: Send + Sync {
```

Comment:
i don't think we make use of async for storage anyway so better to not have these functions return an `impl Storage`

Reply:
> better to not have these functions return an impl Storage

Do you mean `impl Future`?

`Storage` might not, but I like to design my ports pretending I don't know what the impl will be.

Reply:
yeah, `impl Future`, this can be removed if at the end of this integration we dont use async for the BlockAggregatorDB